internal/cmd/logout_test.go

```diff
@@ -117,6 +122,16 @@ func TestLogout(t *testing.T) {
 					UserID:        1,
 					SkipTLSVerify: true,
 					Current:       true,
+					Expires:       api.Time(time.Now().Add(time.Hour * 2).UTC().Truncate(time.Second)),
+				},
+				{
+					Name:          "user2",
+					Host:          srv.Listener.Addr().String(),
+					AccessKey:     "the-access-key",
+					UserID:        2,
+					SkipTLSVerify: true,
+					Current:       true,
```

Comment:
I don't think logout calls `mustBeLoggedIn`, so the test case won't cover the change you are making.

Reply:
logout calls mustbeLoggedIn, and only prints out a success message if logged in, and nothing if logged out (or expired). 

The test case `expired session fails` has two sessions - one is logged in but api call fails (so api count +1), the other is expired so it won't even call the api (hence api count stays as 1).